meson.build

```diff
@@ -48,10 +48,74 @@ targets = []
 
 cc = meson.get_compiler('c')
 fs = import('fs')
+
+# Find the picolibc name for the host cpu. Provide
+# for some common aliases
+
+cpu_family_aliases = {
+		       # aarch64
+		       'arm64' : 'aarch64',
+		       # cris
+		       'crisv32' : 'cris',
+		       # i386
+		       'i486' : 'i386',
+		       'i586' : 'i386',
+		       'i686' : 'i386',
+		       'x86' : 'i386',
+		       # m68hc11
+		       'm6811' : 'm68hc11',
+		       'm6812' : 'm68hc11',
+		       'm68hc12' : 'm68hc11',
+		       # m68k
+		       'fido' : 'm68k',
+		       # m88k
+		       'm88110' : 'm88k',
+		       # or1k
+		       'or1knd': 'or1k',
```

Comment:
I guess I'm not familiar enough about the relation between picolibc and newlib?  I don't see the point of having aliases we don't support.  You said in #196 
> We could leave the archs which have code ready to port in a separate list and let people know when they've selected an arch which just hasn't had picolibc build support added?

Does it mean we have _code ready to port_ for these arches ?  Sorry for my ignorance.

Reply:
Yes, you can see in newlib/libc/machine how many cpus are supported that we don't have build bits for. I figure that we can add the build bits when someone manages to test the result.

Reply:
Hrm. Could just check for newlib/libc/machine/<arch>/meson.build instead of using an array. That's what support means, after all -- you need setjmp/longjmp for the target.